apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/EndringBrevData.kt

```diff
@@ -117,15 +117,19 @@ data class UtAvFengselBrevdata(
 
 data class YrkesskadeRevurderingBrevdata(
     val utbetalingsinfo: Utbetalingsinfo,
+    val stoenadHarOekt: Boolean,
     val yrkesskadeErDokumentert: Boolean,
     val virkningsdato: LocalDate
 ) : EndringBrevData() {
 
     companion object {
         fun fra(behandling: Behandling): YrkesskadeRevurderingBrevdata {
+            val stonadHarOekt = behandling.utbetalingsinfo.beloep.value >
+                behandling.forrigeUtbetalingsinfo!!.beloep.value
```

Comment:
Kanskje se på trygdetiden, og ikke utbetalingensinfoen? Siden du kan havne i en slik situasjon:

```
Forrige iverksatte                                |-> 2000 kr
Yrkesskade revurdering:  |-> 1900 kr ------------ |-> 2200 kr 
```
pga. regulering / søskenjustering / annet, og si at vi _ikke_ øker når vi gjør det. Eller motsatt for den sags skyld, at vi sier vi øker når vi ikke gjør det, siden vi sammenligner med en periode med annen søskenjustering eller institusjonsopphold